deepface/commons/functions.py

```diff
@@ -70,7 +71,21 @@ def load_image(img):
 
     # ---------------------------
 
-    if base64_img is True:
+    filename, file_extension = os.path.splitext(img)
```

Comment:
this logic should be done in `exact_image is not True` block

Reply:
No problems. Before I update the code, is this the format you are looking for ?

```
    elif exact_image is not True:  # image path passed as input
        if os.path.isfile(img) is not True:
            raise ValueError(f"Confirm that {img} exists")

        if (file_extension.lower() == ".heic"):
            ...
        else:
            img = cv2.imread(img)

```